src/charm.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical
+# See LICENSE file for licensing details.
+
+"""SSSD Operator Charm."""
+
+import logging
+
+from ops.charm import CharmBase
+from ops.main import main
+from ops.model import ActiveStatus
+from sssd import Sssd
+
+logger = logging.getLogger(__name__)
+
+
+class SssdCharm(CharmBase):
```

Comment:
How about `SSSDCharm`?

```suggestion
class SSSDCharm(CharmBase):
```

Usually when I type out sssd for a class name I just throw caps-lock on :sweat_smile:

Reply:
Sounds good to me, updated!